:PROPERTIES:
- org-mode configuration
#+Latex_class: els-article
#+LANGUAGE:  en
#+OPTIONS:   title:nil author:nil date:nil  H:2 num:nil toc:nil \n:nil @:t ::t |:t ^:t -:t f:t *:t <:t
#+OPTIONS:   TeX:t LaTeX:t skip:nil d:nil todo:t pri:nil tags:not-in-toc
#+EXPORT_SELECT_TAGS: export
#+EXPORT_EXCLUDE_TAGS: noexport
#+LINK_UP:
#+LINK_HOME:
#+XSLT:
#+DRAWERS: LOGBOOK CLOCK HIDDEN PROPERTIES
#+STARTUP: overview
#+STARTUP: noindent
#+bibliography: Collection.bib
#+cite_export: csl 
##+LaTeX_HEADER: \usepackage{lineno}
##+LaTeX_HEADER: \linenumbers
#+LaTeX_HEADER: \usepackage{setspace}
#+LaTeX_HEADER: \onehalfspacing
#+LaTeX_HEADER: \authblk
#+LaTeX_HEADER: \usepackage{pdfpages}
#+LaTeX_header: \usepackage{textpos}
#+LaTeX_header: \usepackage[final]{draftwatermark}
#+LaTeX_HEADER: \usepackage{gensymb}
#+LaTeX_HEADER: \usepackage{amsmath}
#+LaTeX_HEADER: \usepackage{chemfig}
#+LaTeX_HEADER: \setchemfig{atom style={scale=0.45}}
#+LaTeX_HEADER: \usepackage[]{mhchem}
:END:

#+BEGIN_EXPORT LaTeX
\begin{frontmatter}
\title{Biotinidase Activity is Affected by Both Seasonal Temperature and Filter Collection Cards.}
\author[NSO, UoO]{Matthew P.A. Henderson\corref{cor1}}
\ead{mhenderson@cheo.on.ca}
\author[NSO]{Nathan McIntosh}
\author[NSO]{Amy Chambers}
\author[NSO]{Emily Desormeaux}
\author[NSO]{Michael Kowalski}
\author[NSO]{Jennifer Milburn}
\author[NSO, UO]{Pranesh Chakraborty}
\address[NSO]{Newborn Screening Ontario, Children's Hospital of Eastern Ontario}
\address[UoO]{Department of Medicine, University of Ottawa} 
\cortext[cor1]{Corresponding author}
\end{frontmatter}
#+END_EXPORT

* Abstract
This study set out to examine pre-analytical factors affecting the
frequency of positive results in newborn screening for biotinidase
deficiency. This investigation was prompted by an increase in the
annual screen positive rate for biotinidase deficiency in Ontario from
2.65x10^{-4} in 2016 to 6.57x10^{-4} in 2017.

Season and trend decomposition was used to separate seasonality from
an underlying trend in the time series of biotindase activity
measurements for the period 2014-01-12 to 2019-07-27 (n =
798,770). This analysis revealed a marked seasonal effect (winter =
median + \le 17 MRU, summer = mean - \le 20 MRU) and a non-linear
negative trend. Seasonal temperature was correlated with biotinidase
results (Pearson's r = 0.79) but not with the observed negative trend
(Pearson's r = 0.0025). Time series analysis of biotinidase results
grouped by print lot of filter paper revealed that recently printed
filter paper cards inhibit biotinidase and that this inhibition
resolved over time. This study demonstrates that biotindase activity
is inhibited by both increased seasonal temperature and collection on
newly printed filter cards.

* Keywords
newborn screening, biotinidase deficiency, dried blood spots, filter
paper, seasonal variation, enzyme activity
* Introduction

Newborn screening for biotinidase deficiency is based on measurement
of biotinidase activity in dried blood spots. This investigation was
prompted by a marked increase in the annual screen positive rate for
biotinidase deficiency in Ontario from 2.65x10^{-4} in 2016 to
6.57x10^{-4} in 2017 and set out to examine pre-analytical factors
affecting in newborn screening for biotinidase deficiency.

Biotinidase deficiency is an autosomal recessive disorder caused by
absent or markedly deficient activity of biotinidase, a cytosolic
enzyme that liberates biotin from biocytin during the normal
proteolytic turnover of holocarboxylases and other biotinylated
proteins. The absence of appropriate biotin recycling is associated
with secondary alterations in amino acid, carbohydrate, and fatty acid
metabolism (Figure [[fig:cycle]]). Neurological symptoms of biotinidase
deficiency such as lethargy, muscular hypotonia, grand mal and
myoclonic seizures, and ataxia may occur in the neonatal period. Many
affected children also have skin rash, alopecia, developmental delay,
hearing loss, conjunctivitis and visual problems including optic
atrophy. Metabolic acidosis and the characteristic organic aciduria of
multiple carboxylase deficiency is often absent in the early stages of
the disease.

The state of Virginia was the first jurisdiction to perform universal
newborn screening for biotinidase deficiency, beginning in 1984
[cite:@Wolf2015a]. Biotinidase deficiency was added to the recommended
universal screening panel in the United States in 2006 and has been a
target disorder for newborn screening in Ontario since 2007
[cite:@Watson2006;@Gannavarapu2015]. Based on newborn screening
outcome data from 2006, the incidence of profound biotinidase
deficiency in the United States is estimated at 1/80,000 births, and
partial biotinidase deficiency between 1/31,000 and 1/40,000
[cite:@Strovel2017]. A similar study conducted in Ontario found a
combined prevalence of profound and partial biotinidase deficiency at
a higher ratio of approximately 1 in 15,000 [cite:@Gannavarapu2015].

Diagnosis of biotinidase deficiency is based on demonstrating
deficient enzyme activity in serum or plasma. Patients with profound
biotinidase deficiency have less than 10% of mean normal serum
activity [cite:@Strovel2017]. Patients with partial biotinidase
deficiency have 10-30% of mean normal serum activity
[cite:@Strovel2017]. Most pathogenic variants in the BTD gene cause
complete loss or near-complete loss of biotinidase enzyme activity. A
combination of two of these profound biotinidase deficiency alleles
results in profound biotinidase deficiency. The p.D444H variant is the
most common pathological variant observed in newborn screen positive
infants in Ontario [cite:@Gannavarapu2015]. Compound heterozygotes for
the p.D444H pathogenic variant and a profound biotinidase deficiency
allele have profound biotinidase deficiency. Individuals who are
homozygous for the p.D444H pathogenic variant are expected to have
approximately 45-50% of mean normal serum biotinidase enzyme activity
and do not generally require biotin therapy [cite:@Wolf2016].

While avoiding false negative newborn screening results is of
paramount importance in newborn screening, there is recognition that
false positive results are also an adverse outcome of newborn
screening and need to be reduced for the benefit of families and the
healthcare system [cite:@Kwon2000;@Brockow2019;@Karaceper2016]. A
previous observational study examining the effect of season on the
screen positive rate for biotinidase deficiency found no significant
seasonal effect, this may be due to differences in climate,
transportation practices and screening thresholds
[cite:@Thibodeau1993]. Controlled experiments demonstrate that
exposure to high temperature and humidity decreases biotinidase
activity [cite:@Adam2011]. Evidence presented here demonstrates that
increased seasonal temperatures and collection on newly printed filter
cards inhibits biotindase activity resulting in an increased screen
positive rate.



#+BEGIN_SRC  dot :file ./figures/cycle.pdf :cmdline -Kdot -Tpdf
      digraph {
	    edge [fontname="Helvetica-Bold"];
    // substrate and product nodes
	  node [fontname=helvetica,style=filled, color=lightgrey];
	      diet[label= "Dietary Free\nBiotin"];
	      free[label= "Free Biotin"];
	      atp[label="ATP"];
	      apo[label="Apocarboxylases"];
	      amp[label="AMP"];
	      hc[label="Holocarboxylases"];
	      biocytin[label="Biocytin",shape="oval"];
	      lysine[label="Lysine",shape="oval"];

     //enzyme nodes
	  node [shape=point, style=filled, color=black];
	      hcs[];
	      biotinidase[];
	      protease;
     // function nodes
	   node [shape=octagon, style=filled, color=whitesmoke];
	      functions[label="Fatty Acid Metabolism\nGluconeogenesis\nAmino Acid Metabolism"];

     // connections
	      diet -> free[label="Biotin \n transport"];
	      free -> hcs[arrowhead="none"];
	      atp -> hcs[arrowhead="none"];
	      apo -> hcs[arrowhead="none"];
	      hcs -> hc[taillabel="Holocarboxylase \n Synthetase"];
	      hcs -> amp;
	      hc -> functions[style="dotted"];
	      hc -> protease[headlabel="Proteases",arrowhead="none"];
	      protease -> biocytin;
	      biocytin -> biotinidase[arrowhead="none"];
	      biotinidase -> free[taillabel="Biotinidase"];
	      biotinidase -> lysine;
      }
#+END_SRC

#+CAPTION[]: The Biotin Cycle adapted from [cite:@Strovel2017]. Grey ovals contain substrates and products. Labelled black points represent enzymes. Metabolic functions of holocarboxylases are listed in the light grey octagon. 
#+NAME: fig:cycle
#+ATTR_LaTeX: :width .8\textwidth
#+RESULTS:
[[file:./figures/cycle.pdf]]

* Material and Methods
** Observational Study
A data mining approach was used to investigate the effect of
temperature and filter paper lot on the screen positive rate for
biotinidase deficiency. The laboratory information system was queried
for all newborn screening biotinidase activity results from the first
phase of screening in the period 2014-01-12 to 2019-07-27 (n =
798,770). During the study period dried blood spot biotinidase
activity was measured on the SpotCheck Pro platform (Astoria Pacific,
Oregon USA.). Biotinidase activity was reported in microplate response
units (MRU). 903 filter paper was used for dried blood spot sample
collection (EBF, South Carolina, USA).

Daily mean temperature data for weather station 42183 in Barrie
Ontario was downloaded from Environment Canada. Season and trend
decomposition was used to separate seasonality from an underlying
trend in the 4 year time series of biotindase activity results.

** Experimental Study

A set of dried blood spot samples with a range of biotinidase activity
was created to directly compare filter paper lots. To reduce the
endogenous biotinidase activity whole blood was spun down, the buffy
coat was removed. The remaining red blood cell pellet was washed three
times and re-suspended in saline. Untreated blood was titrated into
aliquots of the washed red blood in a serial dilution. Biotinidase
activity in these samples was measured after spotting onto filter
paper lots W152, W161, W162 and W171 and allowing to dry at room
temperature overnight.

** Software
The manuscript was prepared using the Org-mode environment for
literate programming and reproducible research
[cite:@Schulte2012]. The R language for statistical computing was used
for all data analysis with tidyverse packages for data manipulation,
mcr for deming regression, xts and tsa for time series analysis
[cite:@R2020;@tidyverse;@mcr;@xts;@tsa]. R scripts used for data
analysis are available here:
https://github.com/hendersonmpa/biotinidase_filter_paper

* Results
** Time Series Analysis
Dried blood spot galactose-1-phosphate uridylyltransferase (GALT)
enzyme activity is measured on each newborn screening sample to screen
for galactosemia. Previous studies have shown that GALT activity is
affected by seasonal temperature. Time series analysis was performed
to examine trends in biotinidase and activity over the four year study
period. As expected both biotindase and GALT activity demonstrate
seasonal variation in measured activity, as a result the screen
positive rate for biotinidase deficiency and galactosemia increased in
the warm summer months (Figure [[fig:biotpts]] and [[fig:galtpts]]). However,
there was a period in early 2017 when the screen positive rate for
biotinidase deficiency increased despite external temperatures below
zero (Figure [[fig:biotpts]] between the dashed blue lines). There is no
corresponding change in the galactosemia screen positive rate during
this time period (Figure [[fig:biotpts]] between the dashed blue lines).

#+begin_src R :session *R* :results values :exports none :tangle yes
  library("tidyverse")
  library("lubridate")
  library("magrittr")
  library("readxl")
  library("mcr")
  library("xts")
  library("TTR")
  library("RODBC")
  library("xtable")
  library("TSA")
					  #  library("forecast")
  options(warn=-1) ## options(warn=0) to turn back on
  ## Suppress summarise info
  options(dplyr.summarise.inform = FALSE)
  ## options(tibble.width = Inf)
  ## options(tibble.print_max = Inf) 
  today <- as.Date(now())
  source("credentials.r")

  form2lot <- function(letter, number){
    switch(letter,
	   A = {if (number > 715926 && number <= 791000) {
		  return("B6861310")
		} else if (number > 791000 && number <= 866075) {
		  return("B6884210")
		} else if (number > 866075 && number <= 9441150) {
		  return("B6884210b")
		} else if (number > 9441150) {
		  return("B6900911")
		} else {
		  return("A")
		}
	   },
	   B = {if (number <= 16226) {
		  return("B6900911")
		} else if (number > 16226 && number <= 91301) {
		  return("B6911711")
		} else if (number > 91301 && number <= 166376) {
		  return("B6921412")
		} else if (number > 166376 && number <= 241451) {
		  return("B6931512")
		} else if (number > 241451 && number <= 316526) {
		  return("B6941613")
		} else if (number > 316526 && number <= 354101) {
		  return("B6958113")
		} else if (number > 354101 && number <= 429176) {
		  return("B6967813")
		} else if (number > 429176 && number <= 541751 ) {
		  return("B6978414")
		} else if (number > 541751 && number <= 616826 ) {
		  return("B6996714")
		} else if (number > 616826 && number <= 692604 ) {
		  return("B7012515")
		} else if (number > 692604 && number <= 793323 ) {
		  return("B7019815")
		} else if (number > 793323 && number <= 868401 ) {
		  return("W152")
		} else if (number > 868401 && number <= 914685 ) {
		  return("B7052616")
		} else if (number > 914685 && number <= 991436 ) {
		  return("W161")
		} else if (number > 991436 && number <= 999925 ) {
		  return("W161")
		} else {
		  return("B")
		  ##return(paste0(letter,number))
		}
	   },
	   C = {if (number <=30100) { 
		  return("W161")
		}else if (number > 30100 && number <= 105700) {
		  return("B7078017")
		} else if (number > 105700 && number <= 180700) {
		  return("B7087317")
		} else if (number > 180700 && number <= 257871) {
		  return("B7101418")
		} else if (number > 257871){
		  return("B7115218")
		} else {
		  return("C")
					  #return(paste0(letter,number))
		}

	   },
	   "Unknown")}


  biotquery <- "select s.spcextcode1 as accession,
	     a.ansTimeMeasured as measured_time,
	     s.spcExtcode2 as form,
	     sd.sd2GestationAge as ga,
	     sd.sd2Weight as bw,
	     sd.sd2AgeAtCollection as aoc,
	     a.ansvalueplain as result,
	     va.ResultCode as result_code
	     from (select s.specimenid, a.testid, max(answerix) as answerindex
	     from Answer a inner join specimen s on s.SpecimenID = a.SpecimenID
	     where a.TestId = 179 
	     and a.ansStatus = 110
	     and s.spcextcode1 like '[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]'
	     and substring(s.spcextcode1,1,8) between '20140000' and '20200000'
	     and substring(s.spcextcode1,9,1) not in ('4', '7', '8')
	     group by s.specimenid, a.TestId) a1
	     inner join answer a on a1.SpecimenID = a.SpecimenID and a1.AnswerIndex = a.AnswerIX and a1.TestId = a.TestId
	     inner join specimen s on a1.specimenid = s.specimenid
	     inner join vw_Answers va on s.spcExtcode1 = va.AccessionNumber and a.TestId = va.TestID
	     inner join specimendetail2 sd on sd.SpecimenId = va.SpecimenID
	     order by s.spcextcode1"

  ## biotdata <- with_con(biotquery)
  ## write.csv(biotdata, file= paste0("./data/biot_data_", today, ".csv"))
  biotdata <- read.csv("./data/biot_data_2021-12-07.csv", stringsAsFactors = FALSE)
  biotdata$measured_time  <- ymd_hms(biotdata$measured_time)
  biotdata <- na.omit(biotdata)


  galtquery <- "select s.spcextcode1 as accession,
	     a.ansTimeMeasured as measured_time,
	     s.spcExtcode2 as form,
	     sd.sd2GestationAge as ga,
	     sd.sd2Weight as bw,
	     sd.sd2AgeAtCollection as aoc,
	     a.ansvalueplain as result,
	     va.ResultCode as result_code
	     from (select s.specimenid, a.testid, max(answerix) as answerindex
	     from Answer a inner join specimen s on s.SpecimenID = a.SpecimenID
	     where a.TestId = 13 
	     and a.ansStatus = 110
	     and s.spcextcode1 like '[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]'
	     and substring(s.spcextcode1,1,8) between '20140000' and '20200000'
	     and substring(s.spcextcode1,9,1) not in ('4', '7', '8')
	     group by s.specimenid, a.TestId) a1
	     inner join answer a on a1.SpecimenID = a.SpecimenID and a1.AnswerIndex = a.AnswerIX and a1.TestId = a.TestId
	     inner join specimen s on a1.specimenid = s.specimenid
	     inner join vw_Answers va on s.spcExtcode1 = va.AccessionNumber and a.TestId = va.TestID
	     inner join specimendetail2 sd on sd.SpecimenId = va.SpecimenID
	     order by s.spcextcode1"

  ## galtdata <- with_con(galtquery)
  ## write.csv(galtdata, file= paste0("./data/galt_data_", today, ".csv"))
  galtdata <- read.csv("./data/galt_data_2021-12-08.csv", stringsAsFactors = FALSE)
  galtdata$measured_time  <- ymd_hms(galtdata$measured_time)
  galtdata <- na.omit(galtdata)
#+end_src

#+RESULTS:
: org_babel_R_eoe

#+begin_src R :session *R* :results output latex :exports results :tangle yes
  biotdata %>%
    na.omit() %>%
    group_by(year = year(measured_time)) %>%
    filter(year < 2019) %>%
    summarise(n = n(),
	      median = median(result, na.rm = TRUE),
	      pos = length(result_code[result_code == "BIOT-C-01-012"]),
	      rate = pos/n) %>%
    xtable(caption = "Annual biotinidase screen positive rate",
	   label = "tab:biot_year", display = c("d", "d", "d", "f", "d", "e")) %>%
    print(include.rownames = FALSE)
#+end_src

#+RESULTS:
#+begin_export latex
% latex table generated in R 4.0.3 by xtable 1.8-4 package
% Thu Dec 30 13:32:38 2021
\begin{table}[ht]
\centering
\begin{tabular}{rrrrr}
  \hline
year & n & median & pos & rate \\ 
  \hline
2014 & 140620 & 119.87 &  78 & 5.55e-04 \\ 
  2015 & 140812 & 122.93 &  40 & 2.84e-04 \\ 
  2016 & 143361 & 120.25 &  38 & 2.65e-04 \\ 
  2017 & 144524 & 105.31 &  95 & 6.57e-04 \\ 
  2018 & 146365 & 111.90 &  88 & 6.01e-04 \\ 
   \hline
\end{tabular}
\caption{Yearly Biotinidase Screen Positive Rate} 
\label{tab:biot_year}
\end{table}
#+end_export

#+begin_src R :session *R* :results output latex :exports results :tangle yes
  galtdata %>%
      na.omit() %>%
      group_by(year = year(measured_time)) %>%
    filter(year < 2019) %>%
      summarise(n = n(),
		median = median(result, na.rm = TRUE),
		pos = length(result_code[result_code == "GALT-C-01-012"]),
		rate = pos/n) %>%
      xtable(caption = "Annual galactosemia screen positive rate",
	     label = "tab:galt_year", display = c("d", "d", "d", "f", "d", "e")) %>%
      print(include.rownames = FALSE)
#+end_src

#+RESULTS:
#+begin_export latex
% latex table generated in R 4.0.3 by xtable 1.8-4 package
% Thu Dec 30 13:34:24 2021
\begin{table}[ht]
\centering
\begin{tabular}{rrrrr}
  \hline
year & n & median & pos & rate \\ 
  \hline
2014 & 140678 & 8.37 &  20 & 1.42e-04 \\ 
  2015 & 140171 & 7.93 &  12 & 8.56e-05 \\ 
  2016 & 143352 & 8.13 &  21 & 1.46e-04 \\ 
  2017 & 143261 & 8.46 &  14 & 9.77e-05 \\ 
  2018 & 143592 & 8.22 &  13 & 9.05e-05 \\ 
   \hline
\end{tabular}
\caption{Yearly Galactosemia Screen Positive Rate} 
\label{tab:galt_year}
\end{table}
#+end_export

#+begin_src R :session *R* :results output :exports results :tangle yes
  weather <- read.csv("./data/weather/temp.csv", header = FALSE)
  dates <- as.Date(c("2017-01-01","2017-08-31"))
  labels <- c("","")
  events <- xts(labels, dates)

    #weathertemp <- weather[-c(1,2),] # remove first two days to align with biotts
  biotweek <- biotdata %>%
	  group_by(week = date(floor_date(measured_time, unit = "week"))) %>%
	  summarise(n = n(),
		    median = median(result, na.rm = TRUE),
		    mean = median(result, na.rm = TRUE),
		    pos = length(result_code[result_code == "BIOT-C-01-012"])) %>%
	  filter(week >= "2014-01-12") ## start of Spotcheck Pro results

  tempweek <- weather %>%
    select(date= 5, dailymean = 14) %>%
    group_by(week = floor_date(date(date), unit = "week")) %>%
    filter(week >= "2014-01-12" & week <= biotweek$week[length(biotweek$week)]) %>%
    summarise(temp = mean(dailymean, na.rm = TRUE)) %>%
    na.omit()
 #+end_src

 #+RESULTS:

#+begin_src R :session *R* :results output graphics file :file ./figures/biotpts.pdf :exports results  :tangle yes
  biotcombined <- cbind(biotweek$pos, biotweek$median, tempweek$temp)
  biotcombinedts <- xts(biotcombined, biotweek$week)
  plot(biotcombinedts, col = c("firebrick","steelblue4", "gold2"),
	 multi.panel = TRUE, 
	 yaxis.same = FALSE, 
	 yaxis.right = FALSE,
	 main = "Biotinidase")
  addEventLines(events, srt = 90, pos = 2, on = 1, col = "blue", lty = "dashed")
  addLegend(legend.loc = "topright",
	    legend.names = c("screen positive", "median activity", "temperature (C)"),
	    col = c("firebrick","steelblue4", "gold2"),
	    lty = c("solid","solid", "solid"),
	    on = 1)
#+end_src

#+CAPTION[]: Time series of weekly screen positive biotinidase deficiency referrals, median weekly biotinidase activity and mean weekly temperature (\degree{}C). Dashed blue lines flank a period with an elevated screen positive rate. 
#+NAME: fig:biotpts
#+ATTR_LaTeX: :width \textwidth
#+RESULTS:
[[file:./figures/biotpts.pdf]]

#+begin_src R :session *R* :results output graphics file :file ./figures/galtpts.pdf :exports results :tangle yes
  galtweek <- galtdata %>%
    group_by(week = date(floor_date(measured_time, unit = "week"))) %>%
      summarise(n = n(),
		median = median(result, na.rm = TRUE),
		mean = median(result, na.rm = TRUE),
		pos = length(result_code[result_code == "GALT-C-01-012"])) %>%
      filter(week >= "2014-01-12") ## start of Spotcheck Pro results

    galtcombined <- cbind(galtweek$pos, galtweek$median, tempweek$temp)
    galtcombinedts <- xts(galtcombined, galtweek$week)
    plot(galtcombinedts, col = c("firebrick","steelblue4", "gold2"),
	 multi.panel = TRUE, 
	 yaxis.same = FALSE, 
	 yaxis.right = FALSE,
	 main = "GALT")
  addEventLines(events, srt = 90, pos = 2, on = 1, col = "blue", lty = "dashed")
  addLegend(legend.loc = "topright",
	    legend.names = c("screen positive", "median activity", "temperature (C)"),
	    col = c("firebrick","steelblue4", "gold2"),
	    lty = c("solid","solid", "solid"),
	    on = 1)

#+end_src

#+CAPTION[]: Time series of weekly screen positive galactosemia referrals, median weekly GALT activity and mean weekly temperature (\degree{}C).  Dashed blue lines flank a period with an elevated screen positive rate. 
#+NAME: fig:galtpts
#+ATTR_LaTeX: :width \textwidth
#+RESULTS:
[[file:./figures/galtpts.pdf]]

\clearpage

** Season and Trend Decomposition
Season and trend decomposition was used to identify trends in
biotinidase activity after adjustment for seasonal effects (Figure
[[fig:decomp]]). This analysis revealed a marked seasonal effect with
higher activity in the winter (median + \le 17 MRU) and lower activity
in the summer (median - \le 20 MRU) and a non-linear negative trend
during 2017 and 2018 (Figure [[fig:decomp]]). External temperature was
correlated with biotinidase activity (Pearson's r = 0.79) but not with
the observed negative trend (Pearson's r = 0.025).

#+begin_src R :session *R* :results output graphics file :file ./figures/biotdecomp.pdf :exports results  :tangle yes
  ## Decompose
  ts_biotmedian <- ts(biotweek$median, frequency = 52)
  d <- decompose(ts_biotmedian, "additive")
  #und <- unclass(d)
  biotdecomp <- cbind(biotweek$median, unclass(d$trend), unclass(d$seasonal), unclass(d$random))

  biotdts <- xts(biotdecomp, biotweek$week)
  plot(biotdts, col = c("steelblue4", "firebrick", "gold2", "grey50"),
       yaxis.right = FALSE,
       main = "")
  addLegend(legend.loc = "right",
	    legend.names = c("median", "trend", "seasonal", "random"),
	    col = c("steelblue4", "firebrick", "gold2", "grey50"),
	    lty = c("solid","solid", "solid", "solid"),
	    on = 1)
#+end_src

#+CAPTION[]:Decomposition of the median weekly biotinidiase activity time series into seasonal, random and trend components.
#+NAME: fig:decomp
#+ATTR_LaTeX: :width \textwidth
#+RESULTS:
[[file:./figures/biotdecomp.pdf]]

#+begin_src R :session *R* :results output :exports none :tangle yes
cor(d$seasonal,d$x, use = "pairwise.complete.obs")
cor(d$seasonal,d$trend, use = "pairwise.complete.obs")
#+end_src

#+RESULTS:
: [1] 0.7942895
: [1] 0.02514627

\clearpage

** Biotinidase Activity by Filter Paper Lot
Time series analysis of median weekly biotinidase results grouped by
filter paper lot revealed that filter paper cards exhibited a time
dependent inhibition of biotinidase activity, observed in seven filter
paper lots over four years (Figure [[fig:form]]). Due to a time sensitive
change required in the filter paper collection card, lot W161 was put into
circulation soon after printing. Inhibition of biotinidase activity in
this card lot was most pronounced and took over 5 months to resolve
(Figure [[fig:form]], purple line and Table \ref{tab:w161_months}).

#+begin_src R :session *R* :results output graphics file :file ./figures/biotform.pdf :exports results :tangle yes
  biotdata$form_letter <- gsub("([[:upper:]]{1})([[:digit:]]{6})", "\\1", biotdata$form, perl = TRUE)
  biotdata$form_number <- as.numeric(gsub("([[:upper:]]{1})([[:digit:]]{6})", "\\2", biotdata$form, perl = TRUE))
  biotdata$lot <- unlist(mapply(form2lot, biotdata$form_letter, biotdata$form_number, SIMPLIFY = TRUE, USE.NAMES = FALSE))

  formweekm <- biotdata %>%
      group_by(week = date(floor_date(measured_time, unit = "week")), lot) %>%
      summarise(n = n(),
		mean = mean(result, na.rm = TRUE),
		median = median(result, na.rm = TRUE)) %>%
      filter(n > 50 & week >= "2014-01-12")  %>%  ## start of Spotcheck Pro results
      spread(key = lot, value = mean) %>%
      group_by(week) %>%
      summarise_all(funs(na.omit(.)[1])) 
  lots <- c("B6978414","B7012515", "B7019815", "W152", "W161","B7078017","B7087317", "B7101418")
  #legend.names = c("w121", "w141a", "w41b", "w152", "w161", "w162", "w162b", "w171"),
  formslide <- xts(formweekm[, lots], formweekm$week)

  plot(formslide, col = c("black","red", "steelblue4", "gold2", "darkorchid" , "darkorange", "cyan4"),
       multi.panel = FALSE,
       yaxis.right = FALSE,
       main = "",
       ylab = "Median Weekly Biotinidase Activity")

					  #July 2014 B6978414 W121
					  #July 2015 B7012515 W141
					  #July 2016 W152
					  #Jan 2017 161 
					  #July 2017 W162
					  #Junw 2018 7115218 W171

  addLegend(legend.loc = "bottomright", 
	    legend.names = c("w121", "w141a", "w141b", "w152", "w161", "w162", "w162b", "w171"),
	    col = c("black","red", "steelblue4", "gold2", "darkorchid" , "darkorange", "cyan4"),
	    lty = c("solid", "solid", "solid", "solid", "solid", "solid", "solid"),
	    on = 1)

#+end_src

#+CAPTION[]: Median weekly biotinidase activity by filter paper collection card lot. Each lot of filter paper is plotted independently and indicated with a distinct colour.
#+NAME: fig:form
#+ATTR_LaTeX: :width \textwidth
#+RESULTS:
[[file:./figures/biotform.pdf]]

\clearpage

#+begin_src R :session *R* :results output latex :exports results :tangle yes
  formweekm %>%
    select(week, n, median, W161) %>%
    na.omit() %>%
    group_by(Month = month(week, label = TRUE, abbr = FALSE)) %>%
    summarise("Median" = median(median),
	      "Median W161" = median(W161),
	      "Difference" = `Median W161` - `Median`,
	      "% Difference" = (`Median W161` - `Median`) / `Median` * 100) %>%
    xtable(caption = "Median Activity for Filter Card Lot W161",
	   label = "tab:w161_months", digits = 1, display = c("s", "s","f","f","f","f")) %>%
    print(include.rownames = FALSE)
#+end_src

#+RESULTS:
#+begin_export latex
% latex table generated in R 4.0.3 by xtable 1.8-4 package
% Fri Dec 17 14:55:05 2021
\begin{table}[ht]
\centering
\begin{tabular}{lrrrr}
  \hline
  Month & Median & Median W161 & Difference & \% Difference \\ 
  \hline
  February & 137.3 & 57.6 & -79.8 & -58.1 \\ 
  March & 145.2 & 63.8 & -81.4 & -56.1 \\ 
  April & 135.4 & 69.7 & -65.7 & -48.5 \\ 
  May & 128.2 & 74.8 & -53.3 & -41.6 \\ 
  June & 120.2 & 79.4 & -40.8 & -33.9 \\ 
  July & 117.7 & 92.2 & -25.5 & -21.7 \\ 
  August & 111.3 & 102.1 & -9.3 & -8.3 \\ 
  September & 119.0 & 107.5 & -11.5 & -9.7 \\ 
  October & 124.0 & 122.6 & -1.4 & -1.2 \\ 
  November & 135.4 & 137.8 & 2.4 & 1.8 \\ 
  December & 132.3 & 135.7 & 3.3 & 2.5 \\ 
   \hline
\end{tabular}
\caption{Median Activity for All Samples and Filter Card Lot W161} 
\label{tab:w161_months}
\end{table}
#+end_export

** Inhibition of Biotinidase Resolved After 2 Months
Based on the observed inhibition of biotinidase activity with filter
paper lot W161 (Figure [[fig:form]], purple line and Table
\ref{tab:w161_months}) a controlled experiment was conducted to
compare the biotinidase activity measured after spotting samples onto
selected filter paper lots. Whole blood samples that had been treated
to create a serial dilution of biotinidase activity were spotted onto
a new filter lot (W171) and filter paper lots in circulation (W152,
W161, W162). Lot W171 showed a notable negative bias in the first experiment (Figure
[[fig:spmat]]). Inhibition of biotindase activity by filter paper lot W171
resolved after two months of storage in the laboratory (Figure
[[fig:spmat2]]).

#+begin_src R :session *R* :results output graphics file :file ./figures/spmat.pdf :exports results
  comp <- read_excel("./data/BIO Filter paper Study Data_Sept 2017.xls", sheet = 7)
  pairs(~w152+w161+w162+w171, data=comp,
	panel=function(x,y) {
	    points(x,y)
	    abline(lm(y~x), lty="dashed", col="blue")
	    abline(a = 0, b = 1, col = "red")
	},
	main="")
#+end_src

#+CAPTION: Comparison of biotinidase activity (MRU) in samples spotted simultaneously onto filter paper lots w152, w161, w162, w171 at Time 0. linear regression- blue, line of identity- red
#+LABEL: fig:spmat
#+ATTR_LaTeX: width=0.8\textwidth
#+RESULTS:
[[file:./figures/spmat.pdf]]

#+begin_src R :session *R* :results output graphics file :file ./figures/demingw171.pdf :exports results
  ## comp %<>%
  ##     separate(sample, c("lot", "sample")) %>%
  ##     select(-well) %>%
  ##     spread(key = lot, value = value)
  comp.deming <- mcreg(x = comp$w152, y =comp$w171, error.ratio = 1, alpha = 0.05,
                        mref.name = "W152", mtest.name = "W171", sample.names = NULL,
                        method.reg = "Deming", method.ci = "bootstrap",
                        method.bootstrap.ci = "BCa",
                        nsamples = 999, rng.seed = NULL, rng.kind = "Mersenne-Twister", iter.max = 30,
                        threshold = 1e-06, na.rm = FALSE, NBins = 1e+06)

  plot(comp.deming, x.lab = "W151", y.lab = "W171", main= "")


#+end_src

#+CAPTION: Biotinidase activity (MRU) in samples spotted onto filter paper lot W152 v W171 with Deming regression at time 0.
#+LABEL: fig:demingw171
#+ATTR_LaTeX: width=0.8\textwidth
#+RESULTS:
[[file:./figures/spot_demingw171.pdf]]
\clearpage

#+begin_src R :session *R* :results output graphics file :file ./figures/spmat2.pdf :exports results
  comp2 <- read_excel("./data/BIO Filter paper Study Data_Nov 2017.xls", col_types = "numeric", sheet = 7)
  pairs(~w152+w161+w162+w171, data=comp2,
	  panel=function(x,y) {
	      points(x,y)
	      abline(lm(y~x), lty="dashed", col="blue")
	      abline(a = 0, b = 1, col = "red")
	  },
	  main="")
#+end_src

#+CAPTION: Comparison of biotinidase activity (MRU) in samples spotted simultaneously onto filter paper lots w152, w161, w162, w171 at Time 0 + 2 months. linear regression- blue, line of identity- red
#+LABEL: fig:spmat2
#+ATTR_LaTeX: width=0.8\textwidth
#+RESULTS:
[[file:./figures/spmat2.pdf]]

#+begin_src R :session *R* :results output graphics file :file ./figures/demingw171_2.pdf :exports results
  ## comp %<>%
  ##     separate(sample, c("lot", "sample")) %>%
  ##     select(-well) %>%
  ##     spread(key = lot, value = value)
  comp2.deming <- mcreg(x = comp2$w152, y =comp2$w171, error.ratio = 1, alpha = 0.05,
                        mref.name = "W152", mtest.name = "W171", sample.names = NULL,
                        method.reg = "Deming", method.ci = "bootstrap",
                        method.bootstrap.ci = "BCa",
                        nsamples = 999, rng.seed = NULL, rng.kind = "Mersenne-Twister", iter.max = 30,
                        threshold = 1e-06, na.rm = FALSE, NBins = 1e+06)

  plot(comp2.deming, x.lab = "W151", y.lab = "W171", main= "")
#+end_src

#+CAPTION: Biotinidase activity (MRU) in samples spotted onto filter paper lot W152 v W171 with Deming regression at time 0 + 2 months.
#+LABEL: fig:demingw171_2
#+ATTR_LaTeX: width=0.8\textwidth
#+RESULTS:
[[file:./figures/spot_demingw171_2.pdf]]

\clearpage

* Discussion
This study demonstrates that seasonal temperature and filter paper
collection cards affect biotinidase activity. Our results are
consistent with a previous study that used controlled experiments to
demonstrate that storage at high temperature and humidity decrease
biotinidase activity [cite:@Adam2011]. However a previous
observational study examining the effect of season on the screen
positive rate for biotinidase deficiency found no significant seasonal
effect, the discrepancy with our findings may be due to differences in
climate, transportation practices and screening thresholds
[cite:@Thibodeau1993].

Our program has been aware of seasonal variation in biotinidase
activity for some time. It was thought that elevated temperature and
humidity was the cause of the observed annual increase in screen
positive rate for biotinidase deficiency during the summer. A notable
increase in screen positive rate coinciding with the rapid
distribution of new filter paper cards in the winter of 2017 prompted
us to examine the effect of filter paper collection cards on
biotinidase activity. Once we examined the relationship between filter
paper lot and biotinidase activity a clear time dependent relationship
was evident. Unfortunately, because we distribute new lots of filter
paper cards to birthing centres in July the effect of the filter paper
cards on biotinidase activity had previously been attributed to
seasonal variation.

We have shown that the effect of filter paper lot on biotindase
activity decreases over time. Future controlled studies should examine
the role water content in the filter paper and chemicals used in the
printing process have on biotinidase activity. In the meantime our
program is sequestering filter paper collection cards for two months
prior to release to birthing centres for use in dried blood spot
sample collection for newborn screening.
    
* Conclusions

In Ontario, new batches of filter paper collection cards are often
printed and issued to birthing centres once a year in early
July. Introduction of three batches of filter paper cards in a short
period of time caused a pronounced negative trend in biotinidase
activity leading to an increased biotinidase deficiency screen
positive rate in 2017 and 2018. Time series analysis showed that both
increased seasonal temperatures and collection on newly printed filter
cards inhibit biotindase activity.

* Acknowledgments
Funding: None.
* References
#+print_bibliography:

